airbyte-integrations/bases/base-normalization/Dockerfile

```diff
@@ -28,5 +28,5 @@ WORKDIR /airbyte
 ENV AIRBYTE_ENTRYPOINT "/airbyte/entrypoint.sh"
 ENTRYPOINT ["/airbyte/entrypoint.sh"]
 
-LABEL io.airbyte.version=0.1.78
+LABEL io.airbyte.version=0.1.79
```

Comment:
I think this should be a minor bump (`0.2.0`) since it's not strictly backwards-compatible

Reply:
bumped to 0.2.0